packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -891,8 +895,9 @@ export class PipelineEditor extends React.Component<
           } else {
             // in this case, pipeline was last edited in a "old" version of Elyra and
             // it needs to be updated/migrated.
-            showDialog({
-              title: 'Migrate pipeline?',
+            const filename = this.widgetContext.path.split('/').pop();
+            const migrateDialogResult = await showDialog({
+              title: 'Migrate pipeline ' + filename + '?',
```

Comment:
We refer to this as the pipeline, yet display the pipeline file.  I think using just the basename here (no parent directory or extension) would be best, along with quoting of the basename:

`Migrate pipeline 'untitled'?`

or reference it as the pipeline _file_.  Although taking this route should probably lead to an update of pipeline references to 'pipeline _file_' references below.  So I think the former might be best.

Reply:
I agree, this will look cleaner. We already have a basename but it includes the extension. Will push a commit with this suggestion soon